src/OSET-PL-2.1.xml

```diff
@@ -7,7 +7,7 @@
          <crossRef>http://www.osetfoundation.org/public-license</crossRef>
       </crossRefs>
       <titleText>
-         <br>OSET Public License</br>
+         <br />OSET Public License
```

Comment:
Can we drop `<br/>` entirely?  I'd expect `<titleText>` to already mean "on its own line" (among other things).

Reply:
Hmmm.  Not how the tool currently works.  titleText currently is only a hint for matching and does not carry any formatting significance.

A general question is - should XML properties have both matching and formatting semantics or should we keep them as separate tags?  I have not thought through this - we are likely inconsistent on this today.  I would lean to keeping the matching and formatting tags separate for maximum flexibility.

Reply:
> `titleText` currently is only a hint for matching and does not carry any formatting significance.

Then maybe use `<p>` like most of our existing licenses?

```
$ git grep -A1 '<titleText>'
src/AAL.xml:      <titleText>
src/AAL.xml-         <p>Attribution Assurance License</p>
--
src/AAL.xml:      <titleText>
src/AAL.xml-         <p>ATTRIBUTION ASSURANCE LICENSE (adapted from the original BSD license)</p>
--
…
```

> A general question is - should XML properties have both matching and formatting semantics or should we keep them as separate tags?

Do `<titleText>` and `<optional>` have distinct matching semantics?  I'd expect them to be interchangeable.  If we get [explicit header markup][1], we can separate concerns by dropping `<titleText>` in favor of `<optional><h1>`, etc.

[1]: https://github.com/spdx/license-list-XML/pull/446#discussion_r145519436